packages/runtime/test/integration/assets/docmaps-example-biorxiv-01.jsonld.nt

```diff
@@ -0,0 +1,73 @@
+<https://connect.biorxiv.org/docmaps/> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/spar/pwo/Workflow> .
```

Comment:
static asset that is a copy of one in `docmaps-project/docmaps` repo. I'd like eventually for all the different Docmaps auxiliary stuff like this to be shared.